### ISSUE TITLE
Unknown at rule @apply

### ISSUE THREAD

**Issue by zWingz** [fi3ework/vite-plugin-checker#276]
### Describe the bug

there are warning abount tailwind.css rule `@apply`
<img width="534" alt="image" src="https://github.com/fi3ework/vite-plugin-checker/assets/13031838/57de14a2-e910-43ed-96aa-54939aca6542">

### Reproduction

- install tailwind
- create a css class
- use the rule `@apply`

### Expected behavior

there are not warning

### System Info

```shell
System:
    OS: Linux 4.14 Ubuntu 20.04.6 LTS (Focal Fossa)
    Container: Yes
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.20.0 - ~/.nvm/versions/node/v16.20.0/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v16.20.0/bin/yarn
    npm: 8.19.4 - ~/.nvm/versions/node/v16.20.0/bin/npm
    pnpm: 8.7.1 - ~/.nvm/versions/node/v16.20.0/bin/pnpm
  npmPackages:
    vite-plugin-checker: 0.3.4 => 0.3.4
```

### Validations

- [X] Read the [docs](https://github.com/fi3ework/vite-plugin-checker#readme).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
